src/js/wrapper.ts

```diff
@@ -315,6 +322,22 @@ export const NATIVE = {
     return serialized;
   },
 
+  /**
+   * Convert js severity level which has critical and log to more widely supported levels.
+   * @param level
+   * @returns More widely supported Severity level strings
+   */
+  _processLevel(level: Severity): Severity {
+    if (level === Severity.Critical) {
+      return Severity.Fatal;
+    }
+    if (level === Severity.Log) {
+      return Severity.Debug;
+    }
```

Comment:
```suggestion
    if (level === Severity.Log) {
      return Severity.Debug;
    }
    if (level === Severity.Critical) {
      return Severity.Fatal;
    }
```

Reply:
I believe its more common to have the `log` instead of `critical`, so we spare a comparison in most of the cases